apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/IterableExpectationsTest.kt

```diff
@@ -26,10 +28,10 @@ class IterableExpectationsSpec : ch.tutteli.atrium.specs.integration.IterableExp
         private fun getNotToHaveElementsPair() = "${notToHave.name} ${elements::class.simpleName}" to Companion::notToHaveElements
         private fun notToHaveElements(expect: Expect<Iterable<Int>>) = expect notToHave elements
 
-        private fun minFeaturePair() = feature1<Iterable<Int>, o, Int>(Expect<Iterable<Int>>::min).name to ::minFeature
+        private fun minFeaturePair() = feature1(Expect<Iterable<Int>>::min).name to ::minFeature
```

Comment:
I run into compiler ambiguity issues, re-add the type hints (same same for max and last below)